apps/webhooks.md

```diff
@@ -63,6 +65,65 @@ A notification would look like this:
 
 Depending on server traffic, notifications on events might take a few seconds.
 
+### Verify callback requests
+
+Each callback ePages sends to registered webhooks contains a `X-Epages-Hmac-Sha256` header with a signature that can be used to verify that the request is really coming from ePages.
+This signature is calculated by
+
+**signature = encode_Base64( HMAC_SHA256(secret, message) )**
```

Comment:
Can you mark it like this, please: `signature = encode_Base64( HMAC_SHA256(secret, message) )`